webroot/vue.config.js

```diff
@@ -296,6 +306,9 @@ module.exports = {
         plugins: [
             faviconsPlugin,
             stylelintPlugin,
+            // The runtime-only version of vue-i18n conflicts with test-runner, but is required for built server versions due to CSP.
```

Comment:
What is test-runner?

Reply:
In this case, it's Mocha + Vue Test Utils. Usually this isn't an issue because libraries either don't have a separate runtime-only version, or if they do then it's trivially handled by Webpack. The vue-i18n library is a one-off in that it needs really specific build / plugin handling. They claim to have addressed this in their latest Beta version, but we're not ready to move to that yet.